notebooks/data-sources/TestGrid/metrics/metric_visualization_generic.ipynb

```diff
@@ -0,0 +1,1309 @@
+{
```

Comment:
<p>small nit: can the graphs with overlapping date labels be rotated for readability?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
updated :+1: